consensus/core/src/transaction_certifier.rs

```diff
@@ -414,6 +405,15 @@ impl CertifierState {
             }
         }
 
+        if !certified_blocks.is_empty() {
```

Comment:
Do we really need to guard here by vector length? I though that `inc_by(0)` is acceptable , no?

Reply:
Either is ok but I prefer this to avoid exporting the label value at all when it is 0.